### PR TITLE
Add jsonschema to pip installs for windows cookbook

### PR DESCRIPTION
**Pull request by emersonknapp**
Requirement for https://github.com/ros2/rosidl/pull/722
Related to https://github.com/ros2/ci/pull/695

See https://github.com/ros2/rosidl/pull/722#discussion_r1120569055

### CODE REVIEW

**Review by nuclearsandwich — APPROVED** [review]

**Comment by nuclearsandwich** [comment]
I'm going to open a ros2/ci PR which incorporates this change in order to test that things install as expected before merging.

**Review by clalancette — APPROVED** [review]
Looks good to me.  @nuclearsandwich I'll wait for you to merge this and update https://github.com/ros2/ci/pull/696.

**Pull request merged**
